docusaurus/docs/users/howto/self-hosting.md

```diff
@@ -4,41 +4,55 @@ sidebar_position: 5
 
 ## Creating a Server
 
 In your Hetzner dashboard, navigate to the "Create a server" screen. You will need to indicate a choice in each of the following categories:
 * Server name
-* Region *(Up to you!)*
-* Image/Operating system *(Our suggestion: Ubuntu)*
-* Virtual CPU type *(BobaBoard doesn't need many CPU resources to run, so you can choose to use only a small number of vCPUs with 4 GB ram.)*
-* Network address options *(Our suggestion: Both Public IPv4 & Public IPv6.)*
-* SSH key(s) *(If you don't already have an OpenSSH key you want to use, [generate one](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).)*
+* Region 
+:::tip
+This is where your new server will live, not where *you* live.
+:::
+* Image/Operating system 
+:::tip
+Our suggestion: Ubuntu.
+:::
+* Virtual CPU type 
+:::tip
+BobaBoard doesn't need much CPU usage during everyday use, but setting it up goes very slowly if you pick the cheapest option. Our suggestion: Pay for a bit of higher-tier CPU use, then drop to a lower tier once you're all set up. All you'd have to redo would be [this step](#getting-files-from-the-new-server).
+:::
+* Network address options 
+:::tip
+Our suggestion: Both Public IPv4 & Public IPv6.
+:::
+* SSH key(s) 
+:::tip
+If you don't already have an OpenSSH key you want to use, [generate one](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent).
+:::
 * Cloud configuration *(see below)*
 
 You can skip choosing anything about SSD volumes, firewalls, automatic backups, server placement groups, or labels.
 
-When you get to the Cloud Configuration section, you will need to enter some brief `YAML` configuration instructions. These instructions will tell the new server what to do when it spins up for the first time. The server's first instructions **must** include your public SSH key—without your public SSH key as part of the initial configuration, you won't be able to use your public key to give the server any further instructions, so you'll be locked out and forced to start over.
+When you get to the Cloud Configuration section, you will need to enter some brief `YAML` configuration instructions. These instructions will tell the new server what to do when it first comes to life. The server's first instructions **must** include your public SSH key—without your public SSH key as part of the initial configuration, you won't be able to use your public key to give the server any further instructions, so you'll be locked out and forced to start over.
 
 Copy and paste the following `YAML` data into the Cloud Config field. Then replace everything between the bracketed quotation marks with your public SSH key. 
```

Comment:
Here's how I'd organize these sections to help make them easier to follow. It's not perfect and might need better wording, but it makes the instruction more skimmable and prepares the reader for what they're about to do.

---
The instructions in the cloud configuration section are needed to install NixOS on your server. They will:

1. Create a special file containing your public SSH key to allow you to connect to your server after NixOS is done installing
2. Run a command to replace your current operating system with NixOS

### Creating your SSH key file

:::danger (remove this from above) 
The server's first instructions must include your public SSH key—without your public SSH key as part of the initial configuration, you won't be able to use your public key to give the server any further instructions, so you'll be locked out and forced to start over.
:::

Copy and paste the following YAML data into the Cloud Config field.  [...]

### Installing NixOS on startup